### ISSUE TITLE
Which repo is the main one is unclear

### ISSUE THREAD

**Issue by dkogan** [BrunoLevy/geogram#11]
Hi. I'm looking to package this (and then alicevision) for Debian. I see this repo and the alicevision one:

  https://github.com/alicevision/geogram/

It would be helpful to know which one is the main repo, and which one is the fork. Alicevision claims that their geogram is version 1.7.9 but your `VORPALINE_VERSION_...` variables indicate that you are version 1.8.1. So is your repository is more recent? Would either one be expected to work as the alicevision dependency? Are the APIs supposed to be stable? I.e. if alicevision can build with geogram 1.7.9, will it also build with geogram 1.8.1?

It would also be very helpful to have release tags, so that we could clearly see which commit, exactly, is "geogram 1.8.1".

Thanks!

**Comment by BrunoLevy** [comment]
Hello,

* This repository is the main one, and alicevision is a fork
* Yes, I will create release tags, you are perfectly right, they are needed.
* The API is supposed to remain (reasonably) stable. There were no breaking changes between 1.7.9 and 1.8.1

Best wishes,
-- Bruno

**Comment by dkogan** [comment]
Thanks for replying so quickly. Can you make a tag in the near future? I will then package that version.

**Comment by Tradias** [comment]
@BrunoLevy Heya, it's been a while now, can you make some time to tag and release 1.8.1? :)

**Comment by nursik** [comment]
@BrunoLevy Hello. It will be nice to have tags and we are waiting for the release 1.8.1😊

**Comment by dkogan** [comment]
Hi. This is a gentle ping. The Debian repos will freeze in preparation for the next release in a month or two. I'm waiting on this to be resolved before I can complete the packaging and upload. Thanks!

**Comment by BrunoLevy** [comment]
Yep, I'll have (a bit) more time to do that, will try to do that this week if everything goes well.

**Comment by BrunoLevy** [comment]
Updated copyright in all source files. I have a couple of small things to commit, then 1.8.2 will be there with the tag.

**Comment by BrunoLevy** [comment]
Pushed tag v1.8.1
Please tell me whether everything is OK.

**Comment by dkogan** [comment]
Thank you! I will look at this in a bit. I'm closing this issue now. if
I find any problems, I'll open a new issue.
